db/seeds/pathways/cs_accelerator.rb

```diff
@@ -20,10 +20,10 @@
 activity = Activity.find_by(stem_course_template_no: 'e1e54959-db12-eb11-a813-000d3a86d545')
 pathway.pathway_activities.find_or_create_by(activity_id: activity.id) if activity
 
-activity = Activity.find_by(future_learn_course_uuid: '645ec51f-0b46-4102-a364-90647057f4f2')
+activity = Activity.find_or_create_by(stem_course_template_no: '83e7b049-a5b6-ed11-b597-0022481b59ce')
```

Comment:
was this meant to be a `find_or_create_by`? pathways should only be created once the activity seeds have been run and activities exist, so it seems better to do nothing here, or better would be to error perhaps? though I appreciate that's probably out of scope now